src/client/components/ProductDetails/ProductDetails.component.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import NumberInput from '../NumberInput/NumberInput.component'
+import ButtonComponent from '../Button/Button.component'
+import './ProductDetails.style.css'
+import defaultImage from '../../assets/images/image01.png'
+
+
+export const ProductDetails = (
+    {
+        imgSource,
+        ProductName,
+        RemainingUnit,
+        Price,
+        productColor,
+        productSize,
+        onClick,
+    }
+) => {
+    const [iconSource, setIconSource] = React.useState(true)
+    const changeHeart = () => setIconSource(!iconSource)
+    // The icon can then be used to favorite the product
+
+    return <div id="product-details">
+        <div className='product-image'>
+            <img src={imgSource} alt='product' />
```

Comment:
Is it somehow possible to add a real `alt` attribute?

I suggest adding `alt` as prop but provide a sensible default like what you have now.